.github/workflows/sonar.yml

```
@@ -37,6 +37,9 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+      - name: Set Environment
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)